scipy/ndimage/src/_ni_measure.pyx

```diff
@@ -0,0 +1,173 @@
+######################################################################
+# Cython version of scipy.ndimage.measurement.find_objects() function.
+######################################################################
+
+cimport cython
+from cython cimport sizeof
+import numpy as np
+cimport numpy as np
+
+np.import_array()
+
+cdef extern from *:
+   ctypedef int Py_intptr_t
+
+ctypedef void (*PyArray_CopySwapFunc)(void *, void *, int, void *)
+
+cdef extern from "numpy/arrayobject.h" nogil:
+    ctypedef struct PyArrayIterObject:
+        np.intp_t *coordinates
+        char *dataptr
+        np.npy_bool contiguous
+
+    ctypedef struct PyArrayObject:
+        int nd
```

Comment:
This is not being used anywhere right now. If there is an advantage in using this over `ndarray.ndim` lets do it, if not we can remove it from here.